x/valset-pref/msg_server_test.go

```diff
@@ -554,11 +529,9 @@ func (suite *KeeperTestSuite) TestRedelegateValidatorSet() {
 					del, _ := suite.App.StakingKeeper.GetDelegation(suite.Ctx, test.delegator, valAddr)
 					suite.Require().Equal(del.Shares, test.expectedShares[i])
 				}
-
 			} else {
 				suite.Require().Error(err)
```

Comment:
Can we add checks that exisitng delegations pertain?

Reply:
hmm so like check if the existing validators amount decreased? I'm a lil confused with what exactly to test

Reply:
no the opposite, if redelegation has failed due to an error, we should not have changed positions for delegations, but the delegation for the account should remain the same. Thus we want to check here if this is the case. Would this make sense?

Reply:
oh yea, to check if the state reverts right. I have confirmed through CLI testing that it reverts!